js/viz/chart.d.ts

```diff
@@ -227,6 +227,13 @@ export interface baseLabelObject {
      * @public
      */
     getBoundingRect(): any;
+    /**
+     * @docid
+     * @publicName shift(x, y)
+     * @return object
```

Comment:
shouldn't we change `object` to a `baseLabelObject` ?

Reply:
I have not seen this in other descriptions. Let's ask @IlyaKhD